changelogs/unreleased/9543-check-postgresql-version-on-server-startup.yml

```diff
@@ -0,0 +1,6 @@
+---
+description: "Check PostgreSQL version on server startup."
+issue-nr: 9543
+issue-repo: inmanta-core
+change-type: patch
+destination-branches: [master, iso8]
```

Comment:
We need to add a changelog entry that clearly mentions that the server will not start when running against a PostgreSQL version that doesn't meet the minimal PostgreSQL version mentioned in the docs.